common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -49,6 +49,15 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  val ABMostPopularDefaultTest = Switch(
+    "A/B Tests",
+    "ab-most-popular-default-test",
+    "Switch to change the default of most popular container to show across the guardian first instead of section",
+    safeState = Off,
+    sellByDate = new LocalDate(2015, 11, 17),
```

Comment:
I'd probably extend this by a few days so you don't have a switch expiring mid hack day!